src/ducks/balance/BalancePanels.jsx

```diff
@@ -43,18 +44,20 @@ class BalancePanels extends React.PureComponent {
 
     return (
       <div className={styles.BalancePanels}>
-        {groupsSorted.map(group => (
-          <GroupPanel
-            key={group._id}
-            group={group}
-            warningLimit={warningLimit}
-            expanded={panelsState[group._id].expanded}
-            checked={panelsState[group._id].checked}
-            switches={panelsState[group._id].accounts}
-            onSwitchChange={onSwitchChange}
-            onChange={onPanelChange}
-            withBalance={withBalance}
-          />
+        {groupsSorted.map((group, i) => (
+          <Delayed key={group._id} delay={i < 2 ? 0 : i * 300}>
```

Comment:
This is very strict to state that the third panel and next ones will by below the fold. This will not be the case in a lot of situations (folded panels, panels with few items, big screens).

I'm afraid it will introduce a weird effect by rendering the first two instantly, then adding next ones after some time. I will test it.

Reply:
Just tested: I think it's weird to have the first two rendering instantly, then the others appearing later :thinking:

Reply:
Now, there is no separation between the first and the others, an animation makes them appear fluidly.